corehq/apps/accounting/migrations/0007_make_subscriber_domain_required.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+from django.db.models import Q
+
+
+def check_for_subscriber_domain(apps, schema_editor):
+    if apps.get_model('accounting', 'Subscriber').objects.filter(
+        Q(domain=None) | Q(domain='')
+    ):
+        raise Exception("There exists a subscriber with no domain.")
```

Comment:
Doesn't the alter field already do something like this?  Like if you migrate without a default value, won't that fail hard if there's a subscriber without a domain?  I remember south behaved something like that, but I'm not sure about django migrations.